### PR TITLE
improvement for lose screen

### PR DESCRIPTION
**Pull request by wiesnerroyal**
- game over screen for main theme
- delay after lose to check the score

### CODE REVIEW

**Comment by wiesnerroyal** [comment]
screen

File: src/main.lua [hunk 1]
```diff
@@ -544,6 +573,9 @@ function updateGame()
 		end -- timer is over timerLimit
 	else
 		refreshNeeded = true
+		if showGameOver then
+			drawGameOver()
+		end
```
Comment (thacuber2a03) [inline src/main.lua]: well it just seems like you shoehorned the game over drawing in the code for ending the game
I could leave it like this... OR I could modify the theme engine to also show a game over popup thing...

File: src/main.lua [hunk 2]
```diff
@@ -451,6 +453,33 @@ function drawTexturedBlock(image, x, y, size)
...
+		GFX.setColor(rectColor)
+		GFX.fillRect(145, 20, 110, 26)
+		GFX.drawText("Game Over", 160, 24, kTextAlignment.center)
+	end
+
+	GFX.setImageDrawMode("copy")
+	PD.wait(gameOverDelay)
+	showGameOver = false
+end
+
```
Comment (thacuber2a03) [inline src/main.lua]: putting it inside the playing arena seems a bit intrusive...

**Comment by thacuber2a03** [comment]
I didn't use it properly

**Comment by wiesnerroyal** [comment]
The code is now in the scene object and can be changed individually for each theme. Other ideas for the style of the game over banner?

Best regards.

**Comment by thacuber2a03** [comment]
> Other ideas for the style of the game over banner?

I thought on putting a banner over the whole screen but that's what people are trying to make me avoid 😅

**Pull request closed without merging**